src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div id="name-request-summary">
+    <v-row id="name-request-info">
+      <v-col>
+        <label>
+          <strong>Name Request</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li class="name-request-title mb-4">
+            <strong>{{ getBusinessIdentifier }}</strong> - {{ getNameRequestDetails.approvedName}}
+          </li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li class="mb-4"> Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>Status: {{ getNameRequestDetails.status }}</li>
+          <li>Condition/Consent: {{ conditionConsent() }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+    <v-row id="name-request-applicant-info">
+      <v-col>
+        <label>
+          <strong>Name Request Applicant</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ getNameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ getNameRequestApplicant.phoneNumber }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { getName } from 'country-list'
+
+// Enums
+import { NameRequestStates } from '@/enums'
+
+// Interfaces
+import { GetterIF, NameRequestDetailsIF, NameRequestApplicantIF } from '@/interfaces'
+
+// Mixins
+import { DateMixin } from '@/mixins'
+
+@Component
+export default class NameRequestInfo extends Mixins(DateMixin) {
+  // Global getters
+  @Getter isEntityType!: GetterIF;
+  @Getter isTypeBcomp!: GetterIF;
+  @Getter isTypeCoop!: GetterIF;
+  @Getter getBusinessIdentifier!: GetterIF;
+  @Getter getNameRequestDetails!: NameRequestDetailsIF;
+  @Getter getNameRequestApplicant!: NameRequestApplicantIF;
+  getName = getName
```

Comment:
You may not need this, since you don't use it in the template. I think if you leave it out, the script can still find it in scope (from the mixin).

Reply:
I thought so as well, but when I remove that. I get:
`114:14 Property 'getName' does not exist on type 'NameRequestInfo'.
    112 |     const postal = this.getNameRequestApplicant.postalCode
    113 |     const country = this.getNameRequestApplicant.countryTypeCode
  > 114 |       ? this.getName(this.getNameRequestApplicant.countryTypeCode) : ''`

Reply:
nvm! I was using `this`. Will fix.